apps/tailwind-components/package.json

```diff
@@ -4,7 +4,7 @@
   "version": "0.1.0",
   "type": "module",
   "scripts": {
-    "lint": "vue-tsc --noEmit",
+    "lint": "nuxt prepare && vue-tsc --noEmit",
```

Comment:
ah, the linter also needs this?

Reply:
Indeed. To quote description:
> * "lint": "nuxt prepare && vue-tsc --noEmit", fixes:
> ```
> > Task :apps:tailwind-components:lint
> error TS5083: Cannot read file '[path_to]/molgenis-emx2/apps/tailwind-components/.nuxt/tsconfig.json'.
> ```

After this error it continues with a list of all kinds of error messages (probably needs the file to recognise things defined in tailwind-components and otherwise things invalid code is present)